src/main/java/org/jabref/gui/groups/GroupTreeController.java

```diff
@@ -253,6 +279,22 @@ private ContextMenu createContextMenuForGroup(GroupNodeViewModel group) {
         menu.getItems().add(new SeparatorMenuItem());
         menu.getItems().add(sortAlphabetically);
 
+        // TODO: Disable some actions under certain conditions
```

Comment:
These points are reminders for smaller things that worked in the Swing version but are not yet implemented in JavaFX. They mostly are concerned with the undo/redo functionality.